SteamProject/MySteam/Steam.cpp

```diff
@@ -7,6 +7,7 @@ void FSteam::RunApp()
 {
 	while (true)
 	{
+		system("cls");
 		int UserOption;
```

Comment:
This comes from the previous review, I know that it isn't new code, but remember that it's better to initialize variables in a single line.

```
int UserOption;
UserOption = /* whatever */;
```
is unnecessary when there is an option to type
```
const int UserOption = /* whatever */;
```
plus the second way allows you to make your value constant.